src/components/about/contact-person.tsx

```diff
@@ -20,64 +20,88 @@ export const ContactPerson: React.FC<ContactPersonProps> = ({
   linkedIn,
 }) => {
   return (
-    <Flex sx={{ flexDirection: 'column', alignSelf: 'center' }}>
-      <Box sx={{ width: '25rem', maxWidth: '25rem' }}>
-        <AspectRatio
-          ratio={459 / 527}
-          sx={{ borderRadius: 6, overflow: 'hidden' }}
-        >
-          <Image
-            src={photoPath}
+    <Flex
+      sx={{
+        flexDirection: 'column',
+        alignSelf: 'center',
+        mb: ['20px', '60px', '60px'],
+      }}
+    >
+      <Grid
+        sx={{
+          display: 'flex',
+          gridTemplateColumns: ['auto', '459px 459px', '459px, 459px'],
```

Comment:
Could you explain this line? I feel like there is pretty heavy use of `px` values in  general, and it seems this line is the center of it all.

Is there really no way to achieve what we want without hard coding so many `px` values (boxes, images, margins, links, grids)?